login-workflow/src/screens/SiteOptionsScreen/types.ts

```diff
@@ -0,0 +1,51 @@
+import { WorkflowCardProps } from '../../components/WorkflowCard/WorkflowCard.types';
+import { ErrorManagerProps } from '../../components/Error';
+import { ReactNode } from 'react';
+
+export type SiteOptionsScreenProps = WorkflowCardProps & {
+    /**
+     * The icon to display in the header
+     */
+    icon?: JSX.Element;
+
+    /**
```

Comment:
do we need messageTitle?: string;

Reply:
As per the design, it doesn't have any message title. So I didn’t add. And `message` prop is reactNode so the adopter can customise the message.